heron/statemgrs/src/python/statemanager.py

```diff
@@ -96,11 +83,13 @@ def is_host_port_reachable(self):
     Returns true if the host is reachable. In some cases, it may not be reachable a tunnel
     must be used.
     """
-    try:
-      socket.create_connection((self.host, self.port), 2)
-      return True
-    except:
-      return False
+    for (host,port) in self.hostportlist:
+      try:
+        socket.create_connection((host, port), 2)
+        return True
```

Comment:
If you are unable to connect, can you please print a log message for the host/port that you are unable to connect. I know it is not there previously - but it will be nice to have.